plugins/extractors/bigquery/bigquery.go

```diff
@@ -44,6 +48,7 @@ var sampleConfig = `
 project_id: google-project-id
 table_pattern: gofood.fact_
 include_column_profile: true
+service_account_base64: ____base64_encoded_service_account____
 service_account_json: |-
```

Comment:
Having both in the example config is confusing. 

How about this:

```suggestion
service_account_json: |- # Or service_account_base64 ____base64_encoded_service_account____
```

Reply:
not sure. don't want the field to be a commented field. we can add comments if needed in sample config... since we can show the precedence order also.

Reply:
> we can add comments if needed in sample config... 

not sure what you mean. This _is_ the sample config, right?

Reply:
Yep. I mean something like this - 

```yaml
project_id: google-project-id
table_pattern: gofood.fact_
include_column_profile: true
# Only one of service_account_base64 / service_account_json is needed. If both are present,  service_account_base64 takes precedence
service_account_base64: ____base64_encoded_service_account____
service_account_json: |-
  {
    "type": "service_account",
    "private_key_id": "xxxxxxx",
    "private_key": "xxxxxxx",
    "client_email": "xxxxxxx",
    "client_id": "xxxxxxx",
    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
    "token_uri": "https://oauth2.googleapis.com/token",
    "auth_provider_x509_cert_url": "xxxxxxx",
    "client_x509_cert_url": "xxxxxxx"
  }
collect_table_usage: false
usage_period_in_day: 7`

Reply:
Yes, let's do that.

nit: _limit the line length to 80 and wrap if possible for readability._